pems-server/build.gradle

```diff
@@ -50,6 +50,11 @@ dependencies {
     compile 'com.google.guava:guava:23.0'
 
     compile 'com.github.joschi:jadconfig:0.13.0'
+    
+    compile 'com.squareup.retrofit2:retrofit:2.3.0'
+    compile 'com.google.code.gson:gson:2.8.2'
+    compile 'com.squareup.retrofit2:converter-gson:2.3.0'
```

Comment:
We can, but would this provide any benefits? It would be replacing one dependency with another.

Reply:
It would be swapping the `convertor-gson` for `converter-jackson` yes, but doing so would mean we could remove the core gson library. The benefits would be more consistent json serialization across the server and less dependencies

Reply:
True, as long as we don't end up using Gson somewhere it would be consistent ;). Done.